yt/visualization/volume_rendering/scene.py

```diff
@@ -505,6 +551,18 @@ def _annotate(self, ax, tf, source, label="", label_fmt=None):
             log_scale=source.log_field,
         )
 
+    def _annotate_multi(self, ax, tf, source, label="", label_fmt=None):
```

Comment:
Right now this private method is only called in one place where all parameters are passed explicitly, so I suggest we drop unused default values
```suggestion
    def _annotate_multi(self, ax, tf, source, label, label_fmt):
```